compiler/rustc_infer/src/infer/error_reporting/mod.rs

```diff
@@ -966,7 +966,8 @@ impl<'a, 'tcx> InferCtxt<'a, 'tcx> {
                 ty::GenericParamDefKind::Type { has_default, .. } => {
                     Some((param.def_id, has_default))
                 }
-                ty::GenericParamDefKind::Const => None, // FIXME(const_generics_defaults)
+                // FIXME(const_generics:defaults)
```

Comment:
```suggestion
                // FIXME(const_generic_defaults)
```
But as @lcnr says, it would be better just to fix this now.

Reply:
I think I wasn't immediately sure how to fix this, because it looks like it was very specific to types, so I punted it a bit, but can look more

Reply:
I've done one iteration on it, but not entirely sure if it's being run, so would be worthwhile to look at this